skytemple_randomizer/randomizer/quiz.py

```diff
@@ -167,11 +171,20 @@ def _generate_fallback_question(answer_len) -> QuizQuestion:
             'answers': [FALLBACK_ANSWER for _ in range(answer_len)]
         }
 
+    def _get_vanilla_questions(self) -> List[QuizQuestion]:
+        if self.config['quiz']['include_vanilla_questions']:
+            with open(os.path.join(data_dir(), 'vanilla_questions.yml')) as f:
+                yaml_obj = strictyaml.load(f.read()).data
+                validate(yaml_obj, QUIZ_QUESTIONS_JSON_SCHEMA)
+                return yaml_obj
+        else:
+            return []
+
 
 if __name__ == '__main__':
     import json
-    with open('/home/marco/dev/skytemple/skytemple/randomizer/skytemple_randomizer/data/default.json') as f:
-        data = json.load(f)
+    with open('/home/marco/dev/skytemple/skytemple/randomizer/skytemple_randomizer/data/default.json') as f_test:
```

Comment:
Any particular reason for renaming this to `f_test`?

Reply:
iirc I did it since otherwese line 176 ( `with open(os.path.join(data_dir(), 'vanilla_questions.yml')) as f:`) would cause a warning for shadowing a variable name from an outer scope. I thought renaming the one from the test script looked better.